mms/model_server.py

```diff
@@ -19,20 +19,38 @@ def start():
     """
     args = ArgParser.mms_parser().parse_args()
     pid_file = os.path.join(tempfile.gettempdir(), ".model_server.pid")
+    pid = None
+    if os.path.isfile(pid_file):
+        with open(pid_file, "r") as f:
+            pid = int(f.readline())
+
     if args.stop is True:
-        if os.path.isfile(pid_file):
-            with open(pid_file, "r") as f:
-                try:
-                    os.kill(int(f.readline()), signal.SIGKILL)
-                except OSError:
-                    print("Model server already stopped")
-            os.remove(pid_file)
+        if pid is None:
+            print("Model server is not currently running.")
```

Comment:
should this print be changed to logging

Reply:
This file is only used for command line. print should be sufficient in this case.